lib/manager/nuget/__fixtures__/msbuild-sdk-files/global.json

```diff
@@ -0,0 +1,9 @@
+{
+  "sdk": {
+    "version": "5.0.302",
+    "rollForward": "latestMajor"
+  },
+  "msbuild-sdks": {
+    "YoloDev.Sdk": "0.2.0"
+  }
+}
```

Comment:
Please add final newlines to fixtures

Reply:
Fixed.

Reply:
nope, i still see missing final newlines
```suggestion
}

```

Reply:
@viceice please check my latest commit, it should fix this problem